array_api_compat/common/_helpers.py

```diff
@@ -81,37 +92,44 @@ def your_function(x, y):
     """
     namespaces = set()
     for x in xs:
-        if _is_numpy_array(x):
+        if is_numpy_array(x):
             _check_api_version(api_version)
             if _use_compat:
                 from .. import numpy as numpy_namespace
                 namespaces.add(numpy_namespace)
             else:
                 import numpy as np
                 namespaces.add(np)
-        elif _is_cupy_array(x):
+        elif is_cupy_array(x):
             _check_api_version(api_version)
             if _use_compat:
                 from .. import cupy as cupy_namespace
                 namespaces.add(cupy_namespace)
             else:
                 import cupy as cp
                 namespaces.add(cp)
-        elif _is_torch_array(x):
+        elif is_torch_array(x):
             _check_api_version(api_version)
             if _use_compat:
                 from .. import torch as torch_namespace
                 namespaces.add(torch_namespace)
             else:
                 import torch
                 namespaces.add(torch)
-        elif _is_dask_array(x):
+        elif is_dask_array(x):
             _check_api_version(api_version)
             if _use_compat:
                 from ..dask import array as dask_namespace
                 namespaces.add(dask_namespace)
             else:
                 raise TypeError("_use_compat cannot be False if input array is a dask array!")
+        elif is_jax_array(x):
+            _check_api_version(api_version)
+            # jax.numpy is already an array namespace, but requires this
+            # side-effecting import for __array_namespace__ and some other
+            # things to be defined.
+            import jax.experimental.array_api as jnp
```

Comment:
Will this import go away at some point? Should we guard against that?

Reply:
It's not true that `jax.numpy` is already an array namespace. `jax.experimental.array_api` is an array namespace, and we hope to make `jax.numpy` an array namespace in the future.

Reply:
OK, I fixed the comment. The question still remains though? Should I add a guard here like

```py
try:
    import jax.experimental.array_api as jnp
except ImportError:
    import jax.numpy as jnp
```

for a future JAX version when `jax.experimental.array_api` gets removed? Or will it never be removed?

Reply:
Yes, I think that's probably a reasonable way to future-proof this.

Reply:
The downside of that particular line of code is it will also pass through jax.numpy for older JAX versions that don't have jax.experimental.array_api. I like to avoid explicit version checks in this library if I can, but maybe that's the best thing to do here. 

Or maybe we can just change the logic to this once JAX starts to remove (deprecates?) the experimental import.